assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -22,25 +20,18 @@ import WaysOfSupport from './components/WaysOfSupport';
 import reducer from './reducers/reducers';
 
 
-// ----- Page Startup ----- //
-
-const participation = pageStartup.start();
-setCountry('GB');
-
-
 // ----- Redux Store ----- //
 
-const intCmp = getQueryParameter('INTCMP', 'gdnwb_copts_bundles_landing_default');
+const store = pageInit(reducer);
+
 
-const store = createStore(reducer, {
-  intCmp,
-});
+// ----- Setup ----- //
 
-store.dispatch({ type: 'SET_AB_TEST_PARTICIPATION', payload: participation });
-let waysOfSupport = <WaysOfSupport />;
+let intCmp = store.getState().common.intCmp;
 
-if (intCmp && inCampaign('baseline_test', intCmp)) {
-  waysOfSupport = '';
+if (!intCmp) {
+  intCmp = 'gdnwb_copts_bundles_landing_default';
```

Comment:
Could this be passed to pageInit instead?

Reply:
If we pass this to pageInit, we should probably add the option of receiving a default value for the other arguments also, if not it would be confusing that some arguments receive a default value and others doesn't.

Reply:
Yeah I considered this. Came to the same conclusion as @svillafe, you'd realistically want it to be all or nothing, as an object. In this case you could do it with the `preloadedState` argument, passing in the whole `common` object.